coffee/chosen.jquery.coffee

```diff
@@ -372,7 +372,10 @@ class Chosen extends AbstractChosen
       else
         this.single_set_selected_text(this.choice_label(item))
 
-      unless @is_multiple && (!@hide_results_on_select || (evt.metaKey or evt.ctrlKey))
+      if @is_multiple && (!@hide_results_on_select || (evt.metaKey or evt.ctrlKey))
+        @search_field.val("")
```

Comment:
Sure thing! We’re effectively always clearing the value — but I’ll move this statement out of the if-block to be more explicit about it.